src/swagger-2.ts

```diff
@@ -27,7 +27,7 @@ const TYPES: { [index: string]: string } = {
 };
 
 function capitalize(str: string) {
-  return `${str[0].toUpperCase()}${str.slice(1)}`;
+  return `${str[0].toUpperCase()}${str.slice(1).replace(' ', '')}`;
```

Comment:
If it’s all the same, maybe this should go at the end of `camelCase` instead? I think camelCase is expecting a non-space anyway.

Something like `.replace(/\s+/g, '')` (I think in your example, only the first space would be replaced, and then it would skip the rest—maybe).

Reply:
Actually, wait—`camelCase` should be replacing spaces with capital letters. 🤔 Why is that failing for `'American Express'`?